MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -178,11 +212,11 @@ extension MMEEventsManager {
         return eventDictionary
     }
     
-    func navigationRerouteEvent(routeController: RouteController) -> [String: Any] {
+    func navigationRerouteEvent(routeController: RouteController, MMEEventType: String = MMEEventTypeNavigationReroute) -> [String: Any] {
         let timestamp = Date()
         
         var eventDictionary = self.addDefaultEvents(routeController: routeController)
-        eventDictionary["event"] = MMEEventTypeNavigationReroute
+        eventDictionary["event"] = MMEEventType
```

Comment:
Is this generic event type intentional?

Reply:
Ah, this isn't a constant, its a function parameter. Can you change the variable name to be `eventType` for clarity?